src/bridge/RippleJSBridge.js

```diff
@@ -463,10 +463,29 @@ const RippleJSBridge: WalletBridge<Transaction> = {
 
   getTransactionAmount: (a, t) => t.amount,
 
-  editTransactionRecipient: (account, t, recipient) => ({
-    ...t,
-    recipient,
-  }),
+  editTransactionRecipient: (account, t, recipient) => {
+    const params = new URLSearchParams(recipient)
+    recipient = recipient.split('&')[0]
+
+    // Extract parameters we may need
+    for (const [key, value] of params.entries()) {
+      switch (key) {
+        case 'dt':
+          t.tag = +value
```

Comment:
I think it would be preferable to use `parseInt(value, 10)` and check for potential NaN too